carma-messenger-ui/website/thirdparty/ros/roslib.min.js

```diff
@@ -1 +1 @@
dfModel":39,"./UrdfSphere":40,"./UrdfTypes":41,"./UrdfVisual":42,"object-assign":3}],44:[function(e,t,i){"use strict";var u=Math.pow(2,32),n=!1;function h(){n||(n=!0,console.warn("CBOR 64-bit integer array values may lose precision. No further warnings."))}var r={64:Uint8Array,69:Uint16Array,70:Uint32Array,72:Int8Array,77:Int16Array,78:Int32Array,85:Float32Array,86:Float64Array},s={71:function(e){h();for(var t=e.byteLength,i=e.byteOffset,n=t/8,e=e.buffer.slice(i,i+t),r=new Uint32Array(e),s=new Array(n),o=0;o<n;o++){var a=2*o,c=r[a],a=r[1+a];s[o]=c+u*a}return s},79:function(e){h();for(var t=e.byteLength,i=e.byteOffset,n=t/8,e=e.buffer.slice(i,i+t),r=new Uint32Array(e),s=new Int32Array(e),o=new Array(n),a=0;a<n;a++){var c=2*a,l=r[c],c=s[1+c];o[a]=l+u*c}return o}};void 0!==t&&t.exports&&(t.exports=function(e,t){var i,n;return t in r?(i=e.byteLength,n=e.byteOffset,new r[t](e.buffer.slice(n,n+i))):t in s?s[t](e):e})},{}],45:[function(e,t,i){i.DOMImplementation=window.DOMImplementation,i.XMLSerializer=window.XMLSerializer,i.DOMParser=window.DOMParser},{}],46:[function(e,t,i){t.exports="undefined"!=typeof window?window.WebSocket:WebSocket},{}],47:[function(e,t,i){t.exports=function(){return document.createElement("canvas")}},{}],48:[function(e,t,i){"use strict";var a=e("canvas"),n=a.Image||window.Image;t.exports=function(e,s){var o=new n;o.onload=function(){for(var e=new a,t=e.getContext("2d"),i=(e.width=o.width,e.height=o.height,t.imageSmoothingEnabled=!1,t.webkitImageSmoothingEnabled=!1,t.mozImageSmoothingEnabled=!1,t.drawImage(o,0,0),t.getImageData(0,0,o.width,o.height).data),n="",r=0;r<i.length;r+=4)n+=String.fromCharCode(i[r],i[r+1],i[r+2]);s(JSON.parse(n))},o.src="data:image/png;base64,"+e}},{canvas:47}],49:[function(t,e,i){try{var n=t("webworkify")}catch(e){n=t("webworkify-webpack")}var r=t("./workerSocketImpl");function s(e){this.socket_=n(r),this.socket_.addEventListener("message",this.handleWorkerMessage_.bind(this)),this.socket_.postMessage({uri:e})}s.prototype.handleWorkerMessage_=function(e){var t=e.data;if(t instanceof ArrayBuffer||"string"==typeof t)this.onmessage(e);else{e=t.type;if("close"===e)this.onclose(null);else if("open"===e)this.onopen(null);else{if("error"!==e)throw"Unknown message from workersocket";this.onerror(null)}}},s.prototype.send=function(e){this.socket_.postMessage(e)},s.prototype.close=function(){this.socket_.postMessage({close:!0})},e.exports=s},{"./workerSocketImpl":50,webworkify:7,"webworkify-webpack":6}],50:[function(e,t,i){var s=e("ws");t.exports=function(t){var i=null;function n(e){e=e.data;e instanceof ArrayBuffer?t.postMessage(e,[e]):t.postMessage(e)}function r(e){t.postMessage({type:e.type})}t.addEventListener("message",function(e){e=e.data;if("string"==typeof e)i.send(e);else if(e.hasOwnProperty("close"))i.close(),i=null;else{if(!e.hasOwnProperty("uri"))throw"Unknown message to WorkerSocket";e=e.uri;(i=new s(e)).binaryType="arraybuffer",i.onmessage=n,i.onclose=r,i.onopen=r,i.onerror=r}})}},{ws:46}]},{},[9]);
```

Comment:
It is hard to see but looks like just ROS1 msgs types were converted to ROS2 here. Just documenting for the record.